python/QC3_Plotter.py

```diff
@@ -0,0 +1,310 @@
+
+import pandas as pd
+import xlrd
+from array import array
+import numpy as np
+import ROOT as r
+
+import argparse
+from optparse import OptionParser
+
+parser = argparse.ArgumentParser()
+parser.add_argument('filename', nargs='+')
+args = parser.parse_args()
+
+par1_Short= array( 'f' )
+par1_Long= array( 'f' )
+Err_par1_Short= array( 'f' )
+Err_par1_Long= array( 'f' )
+errX_Short=array('f')
+errX_Long=array('f')
+Detector_Short=array('f')
+Detector_Long=array('f')
+
+for f in args.filename:
```

Comment:
Let's change this implementation then.

The python scripts we add here should not be doing this (Otherwise you'll have to add t for every new observable instead of adding the functionality once). 

This should be included in the C++ code directly.  You can either make a method in the base class PlotterGeneric (preferred) or in the inherited classes for 1D observables (more tedious but livable).